### DISCUSSION TITLE
ISOs available

### DISCUSSION THREAD

**Discussion by prdm0** [archlinux/archinstall#359] [General]
Dear,

can I consider the ISOs that you make available for an official installation that I will consider on my desktop, that is, through these isos will I be able to update the system again and take advantage of the official upgrades of Arch Linux?

Best regards.

**Comment by Torxed** [comment]
The ISO's attached to release candidates and PR's/commits are offered as an easy way to test this code specifically.
They are not meant to replace or guarantee any future proof or stability - only to test this code repository *(the ISO's are based on releng official ISO configuration but with a custom `.zprofile` that uninstalls the official `archinstall` package and updates with a copy from this github)*.

So the short answer is no, but the long answer is that it probably will work and shouldn't cause any issues - but to be 100% sure, use the official ISO unless you're in a mood to help us test this repository/code :)

**Reply by prdm0** [reply]
Is it possible to have the latest version of archinstall with the latest official ISO? For example, is it possible to do something like `sudo pacman -S archinstall --noconfirm` and then proceed with the installation using archinstall or does it not work?

If it is not possible, I will consider the latest ISO of your project.

Thanks for the project. Your project is very interesting, since it weighs practicality with the beauty of Arch Linux.

**Reply by Torxed** [reply]
> Is it possible to have the latest version of archinstall with the latest official ISO? For example, is it possible to do something like `sudo pacman -S archinstall --noconfirm` and then proceed with the installation using archinstall or does it not work?
> 
> If it is not possible, I will consider the latest ISO of your project.
> 
> Thanks for the project. Your project is very interesting, since it weighs practicality with the beauty of Arch Linux.

Absolutely, you can. I think `pacman -Sy; pacman -S archinstall` should get the latest version even on the official ISO.
I'm not entirely sure it will do a clean update, not sure how the "new" python hooks actually work with removing/re-creating directories and stuff, but should work and you can give it a go :)

To verify:
```
pacman -Sy
pacman -S archinstall
python -c "import archinstall; print(archinstall.__version__)"
```
It should say `2.1.4` *(I skipped using pacman for listing the version just to make sure the python lib got installed as it should)*

**Reply by Torxed** [reply]
And yea, ofc.. This requires that your specific mirrors that you use on your machine (or ISO) has caught up with the release that was just the other day. Some times mirrors take a day or two to sync up :)

**Reply by prdm0** [reply]
Thank you, everything went well.
